.github/workflows/pre-submit.yaml

```diff
@@ -24,3 +24,41 @@ jobs: # jobs to run
 
     - name: Build images
       run: make container-build-community
+
+  scorecard-k8s:
+    name: Test Scorecard Tests # https://sdk.operatorframework.io/docs/testing-operators/scorecard/
+    runs-on: ubuntu-22.04
+    env:
+      # see https://github.com/kubernetes-sigs/kind/tags
+      KIND_VERSION: v0.22.0
+      # see https://hub.docker.com/r/kindest/node/tags for available versions!
+      K8S_VERSION: v1.27.11
+      NMO_VERSION: 0.18.0
```

Comment:
Do we really need NMO version set?
It would really simplify maintanance not having it set :grin: 
(also, isn't this workflow run on each PR? The tested code isn't the specified version)

Reply:
>It would really simplify maintanance not having it set 

Yes, it could as we wouldn't need to modify that after each release if we don't set it.
Maybe it is not so much needed here, as IIUC the Version validation (and replaces field) is not something the scorecard cares about. So it could be avoided if you prefer to.

>isn't this workflow run on each PR?

Correct, would it make sense to set if only when it is run on a feature branch?

Reply:
IMHO it makes sense to set those vars to a non default value in order to test a release-like bundle. But the actual numbers aren't relevant. So it's no maintenance issue in the end, even though I agree that it looks like it 😉

Reply:
good point. So, maybe it is enough to add a comment :+1: